scripts/arm32_ci_script.sh

```diff
@@ -178,13 +180,72 @@ function cross_build_corefx {
     ROOTFS_DIR="$__ARMRootfsMountPath" CPLUS_INCLUDE_PATH=$LINUX_ARM_INCPATH CXXFLAGS=$LINUX_ARM_CXXFLAGS ./build-managed.sh -$__buildConfig -skipTests -BuildPackages=false
 }
 
+# Cross builds corefx using Docker image
+function cross_build_corefx_with_docker {
+    __currentWorkingDirectory=`pwd`
+
+    # Check build configuration and choose Docker image
+    if [ "$__buildArch" == "arm" ]; then
+        # TODO: For arm, we are going to embed RootFS inside Docker image.
+        case $__linuxCodeName in
+        trusty)
+            __dockerImage=" chcosta/dotnetcore:ubuntu1404_cross_prereqs_v1"
+            __runtimeOS="ubuntu.14.04"
+        ;;
+        xenial)
+            __dockerImage=" chcosta/dotnetcore:ubuntu1604_cross_prereqs_v1"
+            __runtimeOS="ubuntu.16.04"
+        ;;
+        *)
+            exit_with_error "ERROR: $__linuxCodeName is not a supported linux name for $__buildArch" false
+        ;;
+        esac
+    elif [ "$__buildArch" == "armel" ]; then
+        # For armel Tizen, we are going to construct RootFS on the fly.
+        case $__linuxCodeName in
+        tizen)
+            __dockerImage=" t2wish/dotnetcore:ubuntu1404_cross_prereqs_v2"
+            __runtimeOS="tizen.4.0.0"
+        ;;
+        *)
+            echo "ERROR: $__linuxCodeName is not a supported linux name for $__buildArch"
+            exit_with_error "ERROR: $__linuxCodeName is not a supported linux name for $__buildArch" false
+        ;;
+        esac
+    else
+        exit_with_error "ERROR: unknown buildArch $__buildArch" false
+    fi
+
+    # Build rootfs
+    __dockerCmd="sudo docker run --privileged -i --rm -v $__currentWorkingDirectory:/opt/corefx -w /opt/corefx $__dockerImage"
+    __buildRootfsCmd="./cross/build-rootfs.sh $__buildArch $__linuxCodeName --skipunmount"
+
+    (set +x; echo "Build RootFS for $__buildArch $__linuxCodeName")
+    $__dockerCmd $__buildRootfsCmd
+    sudo chown -R $(id -u -n) cross/rootfs
+
+    # Cross building corefx with rootfs in Docker
+    __buildNativeCmd="/opt/corefx/build-native.sh -buildArch=$__buildArch -$__buildConfig -- cross $__verboseFlag"
+    if [ "$__buildArch" == "arm" ]; then
+        __buildManagedCmd="./build-managed.sh -$__buildConfig -buildArch=$__buildArch -RuntimeOS=$__runtimeOS"
+    else
+        # TODO-armel: We can use same option to arm, i.e. -buildArch and -RuntimeOS options for armel,
```

Comment:
Please file an issue to track unifying this and add it to the main issue in Core-Setup where everything is being tracked.